<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HysTitleBar</class>
 <widget class="QWidget" name="HysTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>56</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>35</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>56</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="6,1,3,1,2">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbIcon">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/destop/app.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTitle">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;黑体&quot;;
border-image:none;
</string>
       </property>
       <property name="text">
        <string>中船重工七一五研究所</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnHome">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color: rgb(255, 255, 255);
}

*:hover {
color:#4169E1;
}

*:checked {
color:#398af1;
}</string>
       </property>
       <property name="text">
        <string>数据采集</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMap">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color: rgb(255, 255, 255);
}

*:hover {
color:#4169E1;
}

*:checked {
color:#398af1;
}</string>
       </property>
       <property name="text">
        <string>态势生成</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnaly">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color: rgb(255, 255, 255);
}

*:hover {
color: #4169E1;
}

*:checked {
color:#398af1;
}</string>
       </property>
       <property name="text">
        <string>复盘分析</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>36</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color: rgb(255, 255, 255);
}

*:hover {
color: #4169E1;
}

*:checked {
color:#398af1;
}</string>
       </property>
       <property name="text">
        <string>整编导出</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSetting">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/destop/setting.png);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMin">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-image: url(:/destop/minux.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMax">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-image: url(:/destop/max.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-image: url(:/destop/close.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
